src/base/_forms.css

```diff
@@ -141,8 +141,7 @@ input[type="radio"] {
 
 input[type="checkbox"]:checked,
 input[type="radio"]:checked {
-  background: var(--primary);
-
+  background: var(--accent-bg, var(--primary));
```

Comment:
I think just `--primary` might be enough here. My only reason is that it might be best keeping accent variables assciated with the `.accent` class alone.

Reply:
Yes, I agree --accent-bg was a fallback solution. It probably should be --element-bg?